### PR TITLE
network/wifi_manager: get connected AP info

### PR DESCRIPTION
**Pull request by pillip8282**
Add a command to wm_test to get connected AP info.

### CODE REVIEW

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 1]
```diff
@@ -30,38 +30,59 @@
...
+#define PROFILE_USAGE											\
+	"\n set a profile:\n"										\
+	"	 wm_test set [ssid] [security mode] [password]\n"		\
+	"	 security mode examples : open, wep_shared \n"			\
+	"	               wpa_aes, wpa_tkip, wpa_mixed  \n"		\
+	"	               wpa2_aes, wpa2_tkip, wpa2_mixed  \n"		\
+	"	               wpa12_aes, wpa12_tkip, wpa12_mixed  \n"	\
+	"	               (*_ent for enterprise)  \n"				\
+	"\n get a profile:\n"										\
+	"	 wm_test get\n"											\
```
Comment (sangwon03) [inline apps/examples/wifi_manager_sample/wm_test.c:59]: I think get_profile is better than get

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 2]
```diff
@@ -30,38 +30,59 @@
...
+	"\n remove a profile:\n"									\
+	"	 wm_test reset\n\n"
```
Comment (sangwon03) [inline apps/examples/wifi_manager_sample/wm_test.c:61]: I think 'reset' is not good for remove a profile

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 3]
```diff
@@ -30,38 +30,59 @@
...
+
+#define INFO_USAGE								\
+	"\n get current state:\n"					\
+	"	 wm_test mode\n"						\
```
Comment (sangwon03) [inline apps/examples/wifi_manager_sample/wm_test.c:65]: I think get state is better than mode

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 4]
```diff
@@ -30,38 +30,59 @@
...
+	"\n get connection info:\n"					\
+	"	 wm_test info\n"
+
+#define REPEATTC_USAGE													\
+	"\n repeat test of APIs:\n"											\
```
Comment (sangwon03) [inline apps/examples/wifi_manager_sample/wm_test.c:70]: I think get_info is better than info

File: apps/examples/wifi_manager_sample/wm_test.c (hunk 1 above)
Comment (pillip8282) [reply apps/examples/wifi_manager_sample/wm_test.c:59]: your comment isn't related to current PR. I'll fix it next PR.

File: apps/examples/wifi_manager_sample/wm_test.c (hunk 2 above)
Comment (pillip8282) [reply apps/examples/wifi_manager_sample/wm_test.c:61]: your comment isn't related to current PR. I'll fix it next PR.

File: apps/examples/wifi_manager_sample/wm_test.c (hunk 3 above)
Comment (pillip8282) [reply apps/examples/wifi_manager_sample/wm_test.c:65]: your comment isn't related to current PR. I'll fix it next PR.

File: apps/examples/wifi_manager_sample/wm_test.c (hunk 4 above)
Comment (pillip8282) [reply apps/examples/wifi_manager_sample/wm_test.c:70]: your comment isn't related to current PR. I'll fix it next PR.

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 5]
```diff
@@ -776,6 +802,27 @@ void wm_get_stats(void *arg)
 	WM_TEST_LOG_END;
 }
 
+void wm_get_conn_info(void *arg)
+{
+	WM_TEST_LOG_START;
+	wifi_manager_info_s info;
+	wifi_manager_result_e res = wifi_manager_get_info(&info);
+	if (res != WIFI_MANAGER_SUCCESS) {
+		printf("[WM] fail to get connection info(%d)\n", res);
```
Comment (jeongchanKim) [inline apps/examples/wifi_manager_sample/wm_test.c:811]: It means we cannot get the info well. But in the below, we are trying to print the info. I think they will be useless in fail case.

**Pull request merged**